app/router.js

```diff
@@ -6,7 +6,7 @@ var Router = Ember.Router.extend({
 });
 
 export default Router.map(function() {
-  this.resource('matches', function() {
+  this.route('matches', { resetNamespace: true }, function() {
```

Comment:
Do you mean that there's no more `resource` in Ember routes? I couldn't find in the docs this... can you share the link saying that it's deprecated?